### PR TITLE
[0.9] Updated changelog for 0.9

### LINKED ISSUE #1508: Write changelog for v0.9

### LINKED ISSUE #1509: Fix shadow example sampling to use LOD=0
**Description**
It's going to be failing with a uniformity analysis error. Needs to use `textureCompareLevel`.

**Repro steps**
Run it with latest Naga.

**Expected vs observed behavior**
No panics!

**Extra materials**
**Platform**

### PR DESCRIPTION
**Pull request by cwfitzgerald**
Fixes #1509. Closes #1508. This is targeted against the 0.9 branch and will be forward ported.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
+  - Added `Features::VERTEX_WRITABLE_STORAGE`.
+  - Added `Features::CLEAR_COMMANDS` which allows you to use `cmd_buf.clear_texture` and `cmd_buf.clear_buffer`.
+  - `Buffer::get_mapped_range` can now have a range of zero.
+  - Added shader label to log messages if validation fails.
+  - Fixed output spirv requiring the "kernal" capability.
+  - Fixed segfault due to improper drop order.
+  - Fixed incorrect dynamic stencil reference for Replace ops.
+  - Stopped unconditionally adding cubemap flags when the backend doesn't support cubemaps.
+  - Updated default storage buffer/image limit to `8` from `4`.
+  - Validation: Ensure that if resources are viewed from the vertex stage, they are read only unless `Features::VERTEX_WRITABLE_STORAGE` is true.
```
Comment (kvark) [inline CHANGELOG.md:15]: can we group them, i.e.
```
  - Validation:
    -  A
    -  B
```

Same for "Fixes:" and "Infrastructure:" (or "Internal:")

Comment (cwfitzgerald) [reply CHANGELOG.md]: Sure, fixed!

**Review by kvark — APPROVED** [review]
Wonderful!

**Review by kvark — APPROVED** [review]
bors r+

**Pull request merged**